CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  
 #### Fixes
 
+* [#1440](https://github.com/ruby-grape/grape/pull/1441): Remove pre-2.2.2 rubies from .travis.yml - [@jlfaber](https://github.com/jlfaber).
```

Comment:
While this is technically correct, what we're doing is officially dropping support for Ruby < 2.2, so this should be in the features section and say something like "Supporting Ruby 2.2 or newer". I also think we could use a paragraph in UPGRADING about it.